lib/dradis/plugins/nikto/gem_version.rb

```diff
@@ -9,7 +9,7 @@ def self.gem_version
       module VERSION
         MAJOR = 3
         MINOR = 8
-        TINY = 0
+        TINY = 1
```

Comment:
This is probably a major change, as it breaks compatibility with the previous functionality.

However that doesn't work with our CE plugin versioning scheme, so at least bump the MINOR.

Reply:
we'll also need the accompanying CHANGELOG entry.